ocrs-cli/src/main.rs

```diff
@@ -182,6 +190,10 @@ Options:
 
     Use a custom text recognition model
 
+  -a, --alphabet \"alphabet\"
+
+    Use a custom alphabet for recognition
```

Comment:
```suggestion
    Specify the alphabet used by the recognition model.
```

I think it is important to clarify here that the alphabet needs to match what the recognition model was trained with.